drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
> I just used fragment identifier but maybe we should define it in ApiDocumentation instead?

I think the right way would be to define it in a separate "my-vocab.jsonld" file. This would then also show how people can define their own terms